docs/operating-scylla/rest.rst

```diff
@@ -32,7 +32,7 @@ The following example shows using the Swagger UI with Docker.
 
     .. code:: sh
 
-      docker run --name some-scylla -p 10000:10000 -d scylladb/scylla:5.0.4 --api-address 0.0.0.0
+      docker run --name scylla-6-2-1 -p 10000:10000 -d scylladb/scylla:6.2.1 --api-address 0.0.0.0
```

Comment:
Would we update this every version? I don't see that 6.2.1 is somehow better than 5.0.4.

Reply:
Hi @avikivity. Isn't 5.0.4 better than 6.2.1, because 5.0 is EOL, and 6.2 is still supported?

If you don't want to hard-code the ``6.2.1`` tag, then do you have a better way to ensure that this example uses a recent and supported version? Should we use the ``latest`` tag? Or the tag for the latest major release (``6.2``)?

Reply:
I swapped it out for ``latest``. Thanks!